js/admin.js

```diff
@@ -0,0 +1,31 @@
+jQuery( document ).ready(function($) {
+	$('#check-status').click(function(){
+		$(".theme-check-report").empty();
+		themeCheckRunPHPCS();
+	});
+	function themeCheckRunPHPCS(){
+		$.ajax({
+			url: themeCheck.ajax_url,
```

Comment:
Since you're running it from the backend you don't need to define `ajax_url` via localization, because it's always available, just use `url: ajaxurl,` here